utils/constants.js

```diff
@@ -12,6 +12,17 @@ const ACCOUNT_NAMES = {
   INVOICE: "vtexgame1invoice",
 }
 
+const SKU_DELIVERY_CUSTOMIZATION_ATTACHMENT = "31"
+const SKU_DELIVERY_AND_PICKUP = "35"
+const SKU_PICKUP_1_SLA = "285"
+const SKU_DELIVERY_MULTIPLE_SLA = "289"
+const SKU_DELIVERY_MULTIPLE_SLA_AND_PICKUP_AT_PORTO_ALEGRE = "290"
+const SKU_SCHEDULED_DELIVERY = "291"
+const SKU_SCHEDULED_PICKUP = "296"
+const SKU_PICKUP_1_SLA_AND_DELIVERY_MULTIPLE_SLA = "298"
+const SKU_SCHEDULED_DELIVERY_AND_DELIVERY_MULTIPLE_SLA = "299"
+const SKU_PICKUP = "312"
```

Comment:
Just curious: Why creating multiple constants instead of a single object with multiple keys? You wouldn't need to export each constant individually that way.

Reply:
My first approach was to export each const instead of using module.exports, but since I've changed that, I believe it'll be better to create a `SKU` object.
I'll do it.

Reply:
Done.